src/scripts/main.js

```diff
@@ -1,8 +1,45 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard');
 
+let tableKeys = {};
+
+[...document.querySelectorAll('th')].map(item => {
+  tableKeys[item.innerText.toLowerCase()] = item.innerText.toLowerCase();
```

Comment:
you repeat yourself with `item.innerText.toLowerCase()`, better save it to variable